### ISSUE TITLE
Update SCC Wiki to Detail Changes

### ISSUE THREAD

**Issue by djrotenberg** [CAMH-Scientific-Computing/SCC#20]
The SCC wiki needs to be updated to reflect the recent changes to the system architecture and upgrades.

**Comment by pipitone** [comment]
> The SCC wiki needs to be updated to reflect the recent changes to the system architecture and upgrades. 

Agreed. You may also consider porting the SCC wiki to github to make it easier to control/allow access to the wiki and consolidate admin.

**Comment by djrotenberg** [comment]
NOTE:The SCC wiki will be ported to the Research IT portal (SharePoint) within the next few months.
